doc/sphinx/tutorial/getting-started-with-mir.md

```diff
@@ -0,0 +1,129 @@
+# Getting started with Mir
+
+This tutorial will guide you through Mir's basic functionality. 
+
+By the end of the tutorial, you will install and run a demo application, learn how to use Mir in different environments, and learn about features that Mir provides for Mir-based compositors. 
+
+If you are unfamiliar with Mir, start with this tutorial, and then proceed to a [developer tutorial](write-your-first-wayland-compositor) which will guide you through the process of writing compositors.
```

Comment:
```suggestion
If you are unfamiliar with Mir, start with this tutorial, and then proceed to a [developer tutorial](write-your-first-wayland-compositor) which will guide you through the process of writing a Mir compositor.
```

Reply:
I don't think we need to repeat "Mir"...

```suggestion
If you are unfamiliar with Mir, start with this tutorial, and then proceed to a [developer tutorial](write-your-first-wayland-compositor) which will guide you through the process of writing a compositor.
```